internal/cmd/login.go

```diff
@@ -105,10 +92,30 @@ func login(cli *CLI, options loginCmdOptions) error {
 		return err
 	}
 
-	if options.Server == "" {
-		if options.NonInteractive {
+	var password string
+	if p, ok := os.LookupEnv("INFRA_PASSWORD"); ok {
+		password = p
+	}
```

Comment:
Not a blocker, but I wonder if this should be next to where we read `INFRA_SERVER`, so that all the env var reading is in the same place.

Reply:
Good catch thx!